packages/create-block/index.ts

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const prompts = require('prompts');
+const path = require('path');
+const spawn = require('cross-spawn');
+const commandLineArgs = require('command-line-args');
+const commandLineUsage = require('command-line-usage');
+
+type Options = {
+  name: string;
+  alias?: string;
+  type: StringConstructor | BooleanConstructor;
+  defaultValue?: string;
+  defaultOption?: string;
+  description: string;
+};
+
+type LanguageType = 'typescript' | 'javascript';
+
+/**
+ * Define the command line options.
+ */
+const options: Options[] = [
+  {
+    name: 'namespace',
+    alias: 'n',
+    type: String,
+    defaultValue: 'create-block',
+    defaultOption: 'create-block',
+    description: 'The namespace for the block. (default: create-block)',
+  },
+  {
+    name: 'blocksDir',
+    alias: 'b',
+    description: 'The directory where the blocks will be created relative to the current working directory. (default: blocks)',
+    type: String,
+    defaultValue: 'blocks',
+    defaultOption: 'blocks',
+  },
+  {
+    name: 'blockLanguage',
+    alias: 'l',
+    description: 'The language for the block. Accepts `typescript` or `javascript`. (default: typescript)',
```

Comment:
So the `--blockLanguage` or `-l` if not passed it will allow the prompt to show by checking this value, so if the command line arg is undefined the prompt will show. The default value there is `typescript` allowing the user to just hit enter and move on.

Reply:
OK, I see, the defaults here are in case the option isn't specified on the command-line and you want to apply a default without prompting the user. Makes sense.

Reply:
As this relates to the command line however I will remove the default language.